### ISSUE TITLE
Upstream is active again (July 2024)

### ISSUE THREAD

**Issue by ChillerDragon** [teeworlds-community/teeworlds#118]
Oy returned with a bunch of merges today. So starting today upstream is the newer and superior code base. And the recommended option to fork of.

I don't think its time to shutdown teeworlds-community just yet since Oy did not return to irc and is probably leaving for another year of inactivity. I would make the bold claim that this community project helped motivate a few devs to send pull requests that got in the official code base this merge cycle.

There is no rush to keep teeworlds-community up to date with the official repo. Everyone can just use the official repo. As the macOS pipeline is failing the first priority should be to fix https://github.com/teeworlds-community/teeworlds/issues/109 so the merges can happen again without any admin intervention. Then we could reset the community branch with a git hard reset if not a single person has a fork of teeworlds-community yet that would be nice to keep the diff cleaner. Since this should not grow to be some kind of competitor project or hard fork.

**Comment by jtbx** [comment]
I agree, I think a reset would be fine. If anyone has any local clones they can just rebase their stuff on top of the reset branch.

Are you sure Oy is actually aware that you want to talk in IRC? If he does not respond for a longer period of time after this it could potentially pay to hard fork teeworlds. Granted most of the changes that were applied in the community fork are now in teeworlds master so a hard fork wouldn't change much.

**Comment by ChillerDragon** [comment]
Yes I am pretty sure he knows that there is some demand of him joining a chat to talk about the future of teeworlds. Around 3 people sent him mails since the last time he was on irc.
